### DISCUSSION TITLE
Loading screen until map ready

### DISCUSSION THREAD

**Discussion by xbattlax** [liodali/osm_flutter#324] [Q&A]
Hi, 
My goal is to implement a loading screen when the map is not ready but I don't understand how to get this information, 

I get it in the widget : 
`OSMFlutter(
          controller:controller,
          trackMyPosition: false,
          initZoom: 12,
          minZoomLevel: 8,
          maxZoomLevel: 14,
          onMapIsReady: (isReady) {
            if (isReady) {
              print("map is ready");
            }
          },`

but I would like it before, to do some test, 

Any idea how to do it ? 

Thank you in advance

**Comment by liodali** [comment]
you can use `mapIsLoading` to put the widget that you want to show as loadingWidget 
it will be hide it automatically
